inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplIntegrationTest.java

```diff
@@ -198,6 +200,13 @@ public void updateKnowledgeBase_WithValidValues_ShouldUpdateKnowledgeBase() {
         kb.setReadOnly(true);
         kb.setEnabled(false);
         kb.setBasePrefix("MyBasePrefix");
+        ValueFactory vf = SimpleValueFactory.getInstance();
+        IRI rootConcept1 = vf.createIRI("http://www.ics.forth.gr/isl/CRMinf/I1_Argumentation");
+        IRI rootConcept2 = vf.createIRI("file:/data-to-load/07bde589-588c-4f0d-8715-c71c0ba2bfdb/crm-extensions/F10_Person");
```

Comment:
Where does this `file:/...` thing come from?

Reply:
I used https://collection.britishmuseum.org/resource/sparql to find concepts for testing.

Reply:
Does the test depend on the BM SPARQL service?